spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/AboutController.java

```diff
@@ -270,7 +270,6 @@ private String repoSelector(String version) {
 		final String BUILD_SNAPSHOT_CRITERIA = "BUILD-SNAPSHOT";
 		final String REPO_SNAPSHOT_ROOT = "https://repo.spring.io/libs-snapshot";
 		final String REPO_MILESTONE_ROOT = "https://repo.spring.io/libs-milestone";
-		final String REPO_RELEASE_ROOT = "https://repo.spring.io/libs-release";
```

Comment:
We should keep the `REPO_RELEASE_ROOT` and add the logic in the selector so that if the version end in `.RELEASE`, use the release repository.

Reply:
Updated.  Thanks for the review!